packages/core/test/dialog/dialogTests.tsx

```diff
@@ -192,6 +192,18 @@ describe("<Dialog>", () => {
             // test existence here because id is generated
             assert.notExists(dialog.find(".no-default-if-no-title").hostNodes().prop("aria-labelledby"));
         });
+
+        it("supports ref objects attached to container", () => {
```

Comment:
you didn't call `done` correctly here: since your test assertion is async, you need to use that additional test parameter (just like the button test code I linked earlier)